src/components/process-timestamp-view.js

```diff
@@ -46,39 +46,48 @@ const ProcessTimestampView = ({
 
     const handleTimestampMessage = useCallback(
         async (event) => {
-            const data = JSON.parse(event.data);
+            const data = event;
```

Comment:
So you can directly use event, isn't it ?

Reply:
Yes, the parsing is done by the library

Reply:
I mean, no need to define `const data = event` and use directly `event` as a variable

Reply:
indeed, I wanted to do the minimum changes